go.mod

```diff
@@ -6,18 +6,18 @@ require (
 	github.com/ghodss/yaml v1.0.0
 	github.com/hashicorp/go-version v1.2.0
 	github.com/hashicorp/terraform-plugin-sdk v1.14.0
-	github.com/rancher/rke v1.2.9
+	github.com/rancher/rke v1.2.13
 	github.com/satori/go.uuid v1.2.1-0.20181028125025-b2ce2384e17b
-	github.com/sirupsen/logrus v1.6.0
-	golang.org/x/crypto v0.0.0-20210616213533-5ff15b29337e // indirect
-	gopkg.in/yaml.v2 v2.2.8
-	k8s.io/api v0.20.0
-	k8s.io/apimachinery v0.20.0
-	k8s.io/apiserver v0.20.0
+	github.com/sirupsen/logrus v1.7.0
+	golang.org/x/crypto v0.0.0-20210711020723-a769d52b0f97 // indirect
+	gopkg.in/yaml.v2 v2.4.0
+	k8s.io/api v0.20.10
+	k8s.io/apimachinery v0.20.10
+	k8s.io/apiserver v0.20.10
 	k8s.io/client-go v12.0.0+incompatible
 )
 
 replace (
-	golang.org/x/sys => golang.org/x/sys v0.0.0-20190916202348-b4ddaad3f8a3
-	k8s.io/client-go => k8s.io/client-go v0.20.0
+	golang.org/x/sys => golang.org/x/sys v0.0.0-20210906170528-6f6e22806c34
+	k8s.io/client-go => k8s.io/client-go v0.20.10
```

Comment:
Go modules version should match with the go modules version used at RKE `v1.2.13` https://github.com/rancher/rke/blob/v1.2.13/go.mod

```
...
-	github.com/sirupsen/logrus v1.7.0
+	github.com/sirupsen/logrus v1.6.0
-	golang.org/x/crypto v0.0.0-20210711020723-a769d52b0f97
+	golang.org/x/crypto v0.0.0-20201002170205-7f63de1d35b0
-	gopkg.in/yaml.v2 v2.4.0
+	gopkg.in/yaml.v2 v2.2.8
-	k8s.io/api v0.20.10
+	k8s.io/api v0.20.0
-	k8s.io/apimachinery v0.20.10
+	k8s.io/apimachinery v0.20.0
-	k8s.io/apiserver v0.20.10
+	k8s.io/apiserver v0.20.0
...
-	golang.org/x/sys => golang.org/x/sys v0.0.0-20210906170528-6f6e22806c34
+	golang.org/x/sys => golang.org/x/sys v0.0.0-20190830141801-acfa387b8d69
-	k8s.io/client-go => k8s.io/client-go v0.20.10
+	k8s.io/client-go => k8s.io/client-go v0.20.0
```

Reply:
Ok, synchronized versions